experimental/ubuntu/disco-19.04/R/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM andysprague44/databricksruntime-minimal:19.04
```

Comment:
This will need to be updated to the official image from docker push of 'minimal' below